packages/gatsby/index.d.ts

```diff
@@ -932,32 +933,58 @@ export interface Store {
   replaceReducer: Function
 }
 
-type logMessageType = (format: string, ...args: any[]) => void
-type logErrorType = (message: string, error?: Error) => void
+type LogMessageType = (format: string, ...args: any[]) => void
+type LogErrorType = (errorMeta: string | Object, error?: Object) => void
+
+export type PhantomActivityTracker = {
+  start(): () => void
+  end(): () => void
+  span: Object
+}
+
+export type ActivityTracker = PhantomActivityTracker & {
+  setStatus(status: string): void
+  panic: LogErrorType
+  panicOnBuild: LogErrorType
+}
+
+export type ProgressActivityTracker = Omit<ActivityTracker, "end"> & {
+  tick(increment?: number): void
+  done(): void
+  total: number
+}
+
+export type ActivityArgs = {
+  parentSpan?: Object
+  id?: string
+}
 
 export interface Reporter {
-  stripIndent: Function
+  stripIndent: TemplateTag
```

Comment:
This doesn't seem to be correct?
![Screenshot 2019-11-05 at 02 27 50](https://user-images.githubusercontent.com/419821/68171398-e207e180-ff73-11e9-8b5a-6520b8b16f78.png)

Should this just be:

```suggestion
  stripIndent: (input: string) => string
```
or we could inline parts of https://unpkg.com/browse/@types/common-tags@1.8.0/index.d.ts to get fuller type def?

Reply:
Probably easier to just use `(input: string) => string` as you've suggested I think. Thanks

Reply:
@pieh 
there is also an:

```javascript

/**
 * If you want to strip the initial indentation from the beginning of each line in a multiline string.
 * Important note: this tag will not indent multiline strings coming from the substitutions.
 * If you want that behavior, use the `html` tag (aliases: `source`, `codeBlock`).
 */
 export const stripIndent: TemplateTag;


```